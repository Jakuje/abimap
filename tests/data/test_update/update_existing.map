# This map file was created with smap

EXISTING_1_0_0
{
    global:
        existing_symbol;
    local:
        *;
} ;

