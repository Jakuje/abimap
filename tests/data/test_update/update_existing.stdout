Added:
    symbol

# This map file was updated with smap

EXISTING_1_0_0
{
    global:
        existing_symbol;
        symbol;
    local:
        *;
} ;

